src/components/Registries/RegistriesCompanies/EditCompany/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { BuildRegistryForm } from '../../../../utils/registryFormBuilder';
+import { registriesActions } from '../../../../redux/actions';
+import { registriesSelectors, blockchainSelectors } from '../../../../redux/selectors';
+
+export default function EditCompany() {
+  const { companyId } = useParams();
+  const dispatch = useDispatch();
+  const userWalletAddress = useSelector(blockchainSelectors.userWalletAddressSelector);
+  useEffect(() => {
+    dispatch(registriesActions.getOfficialUserRegistryEntries.call(userWalletAddress));
+  }, [dispatch, registriesActions]);
```

Comment:
```suggestion
  }, [dispatch, userWalletAddress]);
```
`registriesActions` is an imported module, it won't change and I believe eslint knows this and won't complain
`userWalletAddress` though must be in dependencies